src/main/java/Event.java

```diff
@@ -0,0 +1,11 @@
+public class Event extends Task{
+    public String description;
+    public Event(String input) {
+        String[] strArray = input.split("/");
+        this.description = strArray[0].replace("deadline ", "") + ( "(" + strArray[1].replace("from", "from:")) + (strArray[2].replace("to", "to:") + ")");
```

Comment:
It is recommended to have at maximum 120 characters on one line (hard limit). Try to divide it into two lines to keep the number of characters in each line less than 120.